fdbclient/include/fdbclient/CommitTransaction.h

```diff
@@ -123,59 +129,124 @@ struct MutationRef {
 
 	std::string toString() const {
 		std::string checksumStr;
-		if (checksum.present()) {
-			checksumStr = format("checksum: %s ", std::to_string(checksum.get()).c_str());
+		std::string accumulativeChecksumIndexStr;
+		if (this->checksum.present()) {
+			checksumStr = format("checksum: %s ", std::to_string(this->checksum.get()).c_str());
+			if (this->accumulativeChecksumIndex.present()) {
+				accumulativeChecksumIndexStr = format("accumulativechecksumindex: %s ",
+				                                      std::to_string(this->accumulativeChecksumIndex.get()).c_str());
+			}
 		}
-		uint8_t cType = type & ~CHECKSUM_FLAG_MASK;
-		return format("%scode: %s param1: %s param2: %s",
+		uint8_t mType = this->type & ~CHECKSUM_FLAG_MASK;
+		mType = mType & ~ACCUMULATIVE_CHECKSUM_INDEX_FLAG_MASK;
+		return format("%s%scode: %s param1: %s param2: %s",
```

Comment:
This probably will break tools that parse `fdbdecode` output. So a follow-up check/radar is needed. BTW, should we separate these two checksums?

Reply:
Maybe format as "%s %s code:"?